ui/src/app/pages/profile/data-use-agreement-panel.tsx

```diff
@@ -0,0 +1,159 @@
+import {Button} from 'app/components/buttons';
+import {FlexColumn} from 'app/components/flex';
+import {TextInput} from 'app/components/inputs';
+import {Modal, ModalBody, ModalFooter, ModalTitle} from 'app/components/modals';
+import {TooltipTrigger} from 'app/components/popups';
+import {SpinnerOverlay} from 'app/components/spinners';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+import {
+  AoUTitle,
+  dataUseAgreementStyles,
+  IndentedListItem,
+  IndentedUnorderedList,
+  SecondHeader
+} from './data-use-agreement-styles';
+
+
+export const SanctionModal = (props) => {
+  return <Modal width={750}>
```

Comment:
haven't tested this since I couldn't see it open locally.

Reply:
that said, the width=750 bothers me

Reply:
I'm not sure this is used, but don't want to pull it out as a release blocking thing.